peps/pep-0747.rst

```diff
@@ -289,7 +295,7 @@ This explicit syntax serves two purposes. First, it documents the developer's
 intent to use the value as a type form object. Second, static type checkers
 validate that all rules for type expressions are followed::
 
-  x4 = type(int)  # No error, evaluates to "type[int]"
+  x4 = type(int)  # No error, evaluates to "type[type]"
```

Comment:
The expression `type(int)` evaluates to the object `builtins.type` (because that is the type of the class `int`), which does not inhabit `type[int]`.

Actually, I think it might be clearer to use `type(1)` and `type[int]` here instead, will push that update.

Reply:
Good catch.

More accurately, `type(int)` evaluates to `type[type[int]]`, not `type[type]` (which is the same as `type[type[Any]]`).

Reply:
Yes (although I seem to recall noticing that there isn't consistent agreement among type checkers about whether `type` is `type[Any]` or `type[object]`).

In any case, I updated this example to use `type(1)` and `type[int]` instead; it makes the same point without getting into these weeds.

Reply:
> I seem to recall noticing that there isn't consistent agreement among type checkers about whether type is `type[Any]` or `type[object])`. 

The spec is clear on this point. `type` is equivalent to `type[Any]`. Mypy treats the two differently, which means that it's not conformant with the spec in this regard.